### ISSUE TITLE
Idea: generate fake pulsar from PSRFITS folded profile

### ISSUE THREAD

**Issue by mhvk** [mhvk/scintillometry#92]
In general, we'd like to be able to simulate fake pulsars, but there are lots of settings. One way to give all those is to pass on a PSRFITS folded profile, and use, e.g., the frequency and time settings stored in that file.

**Comment by luojing1211** [comment]
That is a great idea. Are we going to use the folded profile as a single pulse?

**Comment by mhvk** [comment]
Yes, that was the idea, to use the folded profile as the template, and add relevant noise.
